### PR TITLE
add quotes around image files

### PR DESCRIPTION
**Pull request by ncaa2018**
images are still broken

### CODE REVIEW

**Review by ncaa2018 — COMMENTED** [review]
got rid of the extra slash

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -21,33 +21,33 @@ <h1>Naomi Amuzie's Portfolio</h1>
           society, and humanity. I love playing/watching volleyball and 
           reading op-eds. Fav TV Shows include Black Mirror, Sherlock, Big
           Little Lies, and The Handmaid's Tale. </p>
-      <img class="resize" src=/images/ADEC36D4-5273-4327-AB8D-2BBD98FA41E2.JPG/>
+      <img class="resize" alt = "selfie/photo of Naomi" src="/images/ADEC36D4-5273-4327-AB8D-2BBD98FA41E2.JPG">
```
Comment (renguyen) [inline portfolio/src/main/webapp/index.html:24]: Nit: change `alt = "text"` to `alt="text"` here and other occurrences in the file to keep things consistent; the other tags like class and src don't have spaces around the equal sign.

**Review by renguyen — APPROVED** [review]
Love all the photos! I just left one minor comment about spacing with HTML tags you can choose to implement or not.

**Review by ncaa2018 — COMMENTED** [review]
removed spaces around alt

**Pull request merged**